### PR TITLE
add use case and include feedback by swcurran

### PR DESCRIPTION
**Pull request by PRieblinger-esatus**

### CODE REVIEW

File: concepts/0003-protocols/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Aries RFC 0003: Protocols
+# Aries RFC 0003: Protocols-x
```
Comment (dhh1128) [inline concepts/0003-protocols/README.md]: What is the meaning of this change? It makes no sense to me.

File: features/0015-acks/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Aries RFC 0015: ACKs
+# Aries RFC 0015: ACKs RULE
```
Comment (dhh1128) [inline features/0015-acks/README.md]: What is the meaning of this change? It makes no sense to me.

File: index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@
 * [0250: Rich Schema Objects](concepts/0250-rich-schemas/README.md) (2019-10-08 &mdash; [`concept`](/tags.md#concept) [`rich-schemas`](/tags.md#rich-schemas))
 
 ## [DEMONSTRATED](README.md#demonstrated)
-* [0015: ACKs](features/0015-acks/README.md) (2019-12-26, [3 impls](features/0015-acks/README.md#implementations) &mdash; [`feature`](/tags.md#feature))
+* [0015: ACKs RULE](features/0015-acks/README.md) (2019-12-26, [3 impls](features/0015-acks/README.md#implementations) &mdash; [`feature`](/tags.md#feature))
```
Comment (dhh1128) [inline index.md]: What is the justification for renaming this RFC?

**Comment by ryjones** [comment]
Please rebase this on top of master.

**Comment by ryjones** [comment]
This commit has an invalid DCO.

**Comment by dhh1128** [comment]
Okay, I will work on the rebasing.

**Comment by ryjones** [comment]
I did a cherry pick, but then I noticed the DCO issue and stuff

**Comment by dhh1128** [comment]
So, rebasing d24b72a on top of 2cd7918 doesn't create something clean. Here is the result: https://github.com/hyperledger/aries-rfcs/pull/397. It has lots of different DCO errors, not just one.

I feel like we are mixing two concerns here. One is a change from @PRieblinger-esatus; the other is changes to automation from @TelegramSam. Can't we separate them by cherry-picking? Sam's automation changes shouldn't be in the same PR as this one.

I am happy to fix this. Big hammers can hit square pegs into round holes if you hit them hard enough. I just don't understand the goal clearly.

**Comment by dhh1128** [comment]
How about I just cherry-pick d24b72a onto master? Are Sam's automation changes already mainstreamed somewhere else, or is this the only copy of them? Why do we have so many test checkins without DCO?

**Comment by ryjones** [comment]
yes, a cherry pick would be better

**Comment by dhh1128** [comment]
This PR has been superseded by #401  (contains the same commit, cherry-picked)

**Pull request closed without merging**